keyboards/splitkb/kyria/rev2/rev2.c

```diff
@@ -0,0 +1,174 @@
+    {{0, 5}, {1, 5}, {2, 5}, {3, 5}, {4, 5}, {5, 5}, {6, 5}, {7, 5}},
+    {{0, 6}, {1, 6}, {2, 6}, {3, 6}, {4, 6}, {5, 6}, {6, 6}, {7, 6}},
+    {{0, 7}, {1, 7}, {2, 7}, {3, 7}, {4, 7}, {5, 7}, {6, 7}, {7, 7}},
+    {{0, 0}, {1, 0}, {2, 0}, {3, 0}, {4, 0}, {5, 0}, {6, 0}, {7, 0}},
+    {{0, 1}, {1, 1}, {2, 1}, {3, 1}, {4, 1}, {5, 1}, {6, 1}, {7, 1}},
+    {{0, 2}, {1, 2}, {2, 2}, {3, 2}, {4, 2}, {5, 2}, {6, 2}, {7, 2}},
+    {{0, 3}, {1, 3}, {2, 3}, {3, 3}, {4, 3}, {5, 3}, {6, 3}, {7, 3}}
+};
+// clang-format on
+#endif
+
+#ifdef RGB_MATRIX_ENABLE
+/* Map physical under glow LEDs for RGB matrix support
+ *  09, 02, xx, xx, 01, xx,                   xx, 11, xx, xx, 12, 19,
+ *  xx, xx, xx, xx, xx, 00,                   10, xx, xx, xx, xx, xx,
+ *  08, 03, 04, xx, xx, xx, 07, xx,   xx, 17, xx, xx, xx, 14, 13, 18,
+ *                  xx, 05, 06, xx,   xx, xx, xx, 16, 15, xx
+ */
+led_config_t g_led_config = { {
+    // Key Matrix to LED Index
+    { NO_LED, NO_LED, NO_LED, 1,      NO_LED, NO_LED, 2,      9     },
+    { NO_LED, NO_LED, 0,      NO_LED, NO_LED, NO_LED, NO_LED, NO_LED },
+    { NO_LED, 7,      NO_LED, NO_LED, NO_LED, 4,      3,      8      },
+    { NO_LED, NO_LED, 6,      5,      NO_LED, NO_LED, NO_LED, NO_LED },
+    { NO_LED, NO_LED, NO_LED, 11,     NO_LED, NO_LED, 12,     19     },
+    { NO_LED, NO_LED, 10,     NO_LED, NO_LED, NO_LED, NO_LED, NO_LED },
+    { NO_LED, 17,     NO_LED, NO_LED, NO_LED, 14,     13,     18     },
+    { NO_LED, NO_LED, 16,     15,     NO_LED, NO_LED, NO_LED, NO_LED },
+}, {
+    // LED Index to Physical Position
+    {77,24}, {63,8}, {21,8}, {21,40}, {35,40}, {63,56}, {77,56}, {91,40}, {7,40}, {7,8},
+    {147,24}, {161,8}, {203,8}, {203,40}, {189,40}, {161,56}, {147,56}, {133,40}, {217,40}, {217,8}
+}, {
+    // LED Index to Flag
+    LED_FLAG_ALL, LED_FLAG_ALL, LED_FLAG_ALL, LED_FLAG_ALL, LED_FLAG_ALL,
+    LED_FLAG_ALL, LED_FLAG_ALL, LED_FLAG_ALL, LED_FLAG_ALL, LED_FLAG_ALL,
+    LED_FLAG_ALL, LED_FLAG_ALL, LED_FLAG_ALL, LED_FLAG_ALL, LED_FLAG_ALL,
+    LED_FLAG_ALL, LED_FLAG_ALL, LED_FLAG_ALL, LED_FLAG_ALL, LED_FLAG_ALL
+} };
+#endif
+
+#ifdef OLED_ENABLE
+oled_rotation_t oled_init_kb(oled_rotation_t rotation) {
+    return OLED_ROTATION_180;
+}
+
+bool oled_task_kb(void) {
+    if (!oled_task_user()) {
+        return false;
+    }
+    if (is_keyboard_master()) {
+        // QMK Logo and version information
+        // clang-format off
+        static const char PROGMEM qmk_logo[] = {
+            0x80,0x81,0x82,0x83,0x84,0x85,0x86,0x87,0x88,0x89,0x8a,0x8b,0x8c,0x8d,0x8e,0x8f,0x90,0x91,0x92,0x93,0x94,
+            0xa0,0xa1,0xa2,0xa3,0xa4,0xa5,0xa6,0xa7,0xa8,0xa9,0xaa,0xab,0xac,0xad,0xae,0xaf,0xb0,0xb1,0xb2,0xb3,0xb4,
+            0xc0,0xc1,0xc2,0xc3,0xc4,0xc5,0xc6,0xc7,0xc8,0xc9,0xca,0xcb,0xcc,0xcd,0xce,0xcf,0xd0,0xd1,0xd2,0xd3,0xd4,0};
+        // clang-format on
+
+        oled_write_P(qmk_logo, false);
+        oled_write_P(PSTR("Kyria rev2\n\n"), false);
```

Comment:
And if we're moving stuff:
```suggestion
        oled_write_P(PSTR("Kyria ", false);
#if defined(KEYBOARD_splitkb_kyria_rev1)
        oled_write_P(PSTR("rev1"), false);
#elif defined(KEYBOARD_splitkb_kyria_rev2)
        oled_write_P(PSTR("rev2"), false);
#endif
        oled_advance_page(true);
        oled_advance_page(true);
```